staff/sergio-luz/js-arrays/push.js

```diff
@@ -0,0 +1,11 @@
+function push() {
+    if( !(arguments[0] instanceof Array)) throw Error('array is not valid');
+    if(typeof arguments[1]==='undefined')  throw Error('element not defined');
+
+    var count = 0;
+    for (var i = 1; i < arguments.length; i++) {
+        arguments[0][arguments[0].length + count] = arguments[i];
+        count++;
+    }
+    return arguments[0];
```

Comment:
according to MDN push returns the array length. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push

Reply:
True! I modified it to check if the array was created well and I forgot to change it